### ISSUE TITLE
Hope to introduce  image style migration and Image Fusion Methods

### ISSUE THREAD

**Issue by jackdiy** [Mikubill/sd-webui-controlnet#404]
I would like to express my gratitude for your outstanding contribution to the field of image generation. Recently, I have used the ControlNet model and have come to appreciate its ability to enhance the continuity and consistency of image generation. This has inspired us to explore better control methods for generating videos. 
Therefore, I would like to propose a new requirement,can we introduce better image style transfer and fusion methods to achieve effects similar to Gen-1 of Runwayml (Video to video)

**Comment by AbyszOne** [comment]
Like what in particular? There are tons of features into this extension to achieve that with proper human input.

**Comment by jackdiy** [comment]
Thanks for your reply, maybe "Image mixer" is a worthy method, but the effect may not be ideal, after all, it does not take into account the continuity of image generation: 
Lambda Demo：https://cloud.lambdalabs.com/demos/ml/image-mixer-demo
https://github.com/justinpinkney/stable-diffusion

**Comment by AbyszOne** [comment]
That is a topic we have discussed at length [here. ](https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/4595#discussioncomment-5107496) 
TLDR. There is no alternative yet to the conceptual grade mix that MJ achieves. However, this extension allows for a pseudo mix that becomes more versatile with each update. I have also tried to give wind to this course, but surely the owner already has enough work and the progress is gradual.
At least major advances are being made in the sequential combination of images for stylized animations.
